run.sh

```diff
@@ -1,8 +1,10 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Bash was easier to get working. Watch out about switching back to sh because if `sh` is not in the appimage, you might accidentally use your systems sh which doesn't have the environment set up so you will get random errors.